src/components/area-plot/AreaPlot.js

```diff
@@ -55,8 +55,8 @@ function PivotedAreaPlot({ plotConfig }) {
       stackId: '1',
       dot: true,
       strokeWidth: 2,
-      stroke: getPaletteColorByIndex(plotConfig, index),
```

Comment:
Is it right that we pass this arg (the data key) in? I saw the function takes it in but we weren't passing it in in most places

Reply:
Move out these color changes, and then confirm that Line and MultiLine still work.